internal/model/subscription_key.go

```diff
@@ -0,0 +1,33 @@
+package model
+
+import (
+	"errors"
+	"strings"
+)
+
+type SubscriptionKey = KeyType[SubscriptionPayload, *SubscriptionPayload]
+
+func NewSubscriptionKey(module, name string) SubscriptionKey {
+	return newKey[SubscriptionPayload](strings.Join([]string{module, name}, "-"))
+}
+
+func ParseSubscriptionKey(key string) (SubscriptionKey, error) {
+	return parseKey[SubscriptionPayload](key)
+}
+
+type SubscriptionPayload struct {
+	Ref string
```

Comment:
This should be structured data not a string blob, that's the whole idea of having a Payload structure and having it parsed. Same with the other ones.

Reply:
updating cron job key as well